app/sequencescape_excel/sequencescape_excel/helpers/worksheet.rb

```diff
@@ -28,6 +27,12 @@ module Worksheet
             color: '00',
             edges: %i[left right top bottom]
           }
+        },
+        string: {
+          format_code: '@'
+        },
+        integer: {
```

Comment:
Integer rendered as such. I haven't applied custom styles to floats, as we'll probably have a range of approaches wrt decimal places and the like.